service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/controller/utils/ReportRequestUtilsTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.nhs.adaptors.oneoneone.cda.report.controller.utils;
+
+import static java.nio.charset.Charset.defaultCharset;
+import static java.nio.file.Files.readAllBytes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.net.URL;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.w3c.dom.Node;
+
+import lombok.SneakyThrows;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01ClinicalDocument1;
+import uk.nhs.itk.envelope.DistributionEnvelopeDocument;
+
+public class ReportRequestUtilsTest {
+    private static final String EFFECTIVE_DATE = "20291030182905+00";
+    private static final String TYPE_ID_EXTENSION = "POCD_HD000040";
+    private static final int VERSION_NUMBER = 1;
+    private static final String RECORD_TARGET_TYPE_CODE = "RCT";
+    private static final String PATIENT_ROLE = "PAT";
+    private static final String ADDRESS_USE = "H";
+    private static final int PAYLOADS_COUNT = 6;
+    private static final String MANIFEST_ID = "uuid_A25CDB08-9AE9-483D-9833-D78A6E40D0AF";
+    private static final String CLINICAL_DOCUMENT_NODE_NAME = "ClinicalDocument";
+
+    private XmlUtils xmlUtils = Mockito.mock(XmlUtils.class);
```

Comment:
You could use `@Mock` annotation here instead of `Mockito.mock(XmlUtils.class);`